src/open_inwoner/openzaak/utils.py

```diff
@@ -143,3 +144,25 @@ def translate_single_status(status_text: str) -> str:
         )
     except StatusTranslation.DoesNotExist:
         return ""
+
+
+def get_fetch_parameters(request) -> dict:
```

Comment:
Good idea to clean this up. Maybe use a name like `get_user_fetch_parameters()` so we keep it for user-related params only?